### PR TITLE
gui/trophy collection: Fix the missing text.

### PR DESCRIPTION
**Pull request by Croden1999**
sorry, my bad

### CODE REVIEW

File: vita3k/gui/src/trophy_collection.cpp [hunk 1]
```diff
@@ -428,7 +428,7 @@ void draw_trophy_collection(GuiState &gui, EmuEnvState &emuenv) {
         draw_list->AddRectFilled(VIEWPORT_POS, BG_POS_MAX, IM_COL32(31.f, 12.f, 0.f, 255.f), 0.f, ImDrawCornerFlags_All);
 
     auto lang = gui.lang.trophy_collection;
-    auto common = gui.lang.common.main;
+    auto common = emuenv.common_dialog.lang.common;
     const auto hidden_trophy_str = common["hidden_trophy"].c_str();
```
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply vita3k/gui/src/trophy_collection.cpp:432]: and go to first emuenv.common_dialog.lang.common in 494 rename it by common and done same for all other

[the remark this answers is not in the document]
Comment (Croden1999) [reply vita3k/gui/src/trophy_collection.cpp]: done

File: vita3k/gui/src/trophy_collection.cpp [hunk 2]
```diff
@@ -491,11 +491,11 @@ void draw_trophy_collection(GuiState &gui, EmuEnvState &emuenv) {
                 ImGui::TextColored(GUI_COLOR_TEXT, "%s", delete_str);
                 ImGui::PopTextWrapPos();
                 ImGui::SetCursorPos(ImVec2((POPUP_SIZE.x / 2) - (BUTTON_SIZE.x + (20.f * SCALE.x)), POPUP_SIZE.y - BUTTON_SIZE.y - (22.0f * SCALE.y)));
-                const auto cancel_str = !emuenv.common_dialog.lang.common["cancel"].empty() ? emuenv.common_dialog.lang.common["cancel"].c_str() : "Cancel";
+                const auto cancel_str = !common["cancel"].empty() ? common["cancel"].c_str() : "Cancel";
                 if (ImGui::Button(cancel_str, BUTTON_SIZE) || ImGui::IsKeyPressed(emuenv.cfg.keyboard_button_circle))
```
Comment (Zangetsu38) [inline vita3k/gui/src/trophy_collection.cpp:495]:
```suggestion
                if (ImGui::Button(common["cancel"].c_str(), BUTTON_SIZE) || ImGui::IsKeyPressed(emuenv.cfg.keyboard_button_circle))
```

File: vita3k/gui/src/trophy_collection.cpp [hunk 3]
```diff
@@ -491,11 +491,11 @@ void draw_trophy_collection(GuiState &gui, EmuEnvState &emuenv) {
                 ImGui::TextColored(GUI_COLOR_TEXT, "%s", delete_str);
                 ImGui::PopTextWrapPos();
                 ImGui::SetCursorPos(ImVec2((POPUP_SIZE.x / 2) - (BUTTON_SIZE.x + (20.f * SCALE.x)), POPUP_SIZE.y - BUTTON_SIZE.y - (22.0f * SCALE.y)));
-                const auto cancel_str = !emuenv.common_dialog.lang.common["cancel"].empty() ? emuenv.common_dialog.lang.common["cancel"].c_str() : "Cancel";
+                const auto cancel_str = !common["cancel"].empty() ? common["cancel"].c_str() : "Cancel";
```
Comment (Zangetsu38) [inline vita3k/gui/src/trophy_collection.cpp:494]: Sorry, delete it, no need check now

Comment (Croden1999) [reply vita3k/gui/src/trophy_collection.cpp]: done

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**